client/src/app/components/labels-from-items/labels-from-items.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { LabelProps } from "@patternfly/react-core";
+import { LabelCustomColor } from "../LabelCustomColor";
+import { useTranslation } from "react-i18next";
+
+interface RandomColorLabelProps extends LabelProps {}
+
+export const RandomColorLabel: React.FC<RandomColorLabelProps> = ({
+  ...props
+}) => {
+  const getRandomColor = () => {
+    const letters = "0123456789ABCDEF";
+    let color = "#";
+    for (let i = 0; i < 6; i++) {
+      color += letters[Math.floor(Math.random() * 16)];
+    }
+    return color;
+  };
```

Comment:
There is an alternative that looks like it can generate some better colors (like keeping super dark and super light colors out):  https://gist.github.com/bendc/76c48ce53299e6078a76